scripts/superdesk-packaging/packaging.js

```diff
@@ -12,11 +12,12 @@
 
     'use strict';
 
-    PackagesService.$inject = ['api', '$q', 'archiveService', 'lock', 'autosave', 'authoring', 'desks'];
-    function PackagesService(api, $q, archiveService, lock, autosave, authoring, desks) {
+    PackagesService.$inject = ['api', '$q', 'archiveService', 'lock', 'autosave', 'authoring', 'authoringWorkspace', 'desks', '$rootScope'];
+    function PackagesService(api, $q, archiveService, lock, autosave, authoring, authoringWorkspace, desks, $rootScope) {
         var self = this;
 
         this.groupList = ['main', 'story', 'sidebars', 'fact box'];
+        this.packageGroupItems = {};
```

Comment:
imo it would be better to somehow reset this on that directive `$destroy` event or when it's opening, it could have old data, like if you would add an item, go somewhere else, somebody would remove it, and you would start editing that highlight again, it would return true that an item is added using this imo